### DISCUSSION TITLE
Add Oracle Linux 8?

### DISCUSSION THREAD

**Discussion by HyptexPvP** [pivpn/pivpn#1525] [Feature requests]
For oracle cloud users, i think it'd be a goood idea.

**Comment by coolapso** [comment]
Doesn't oracle cloud provide debian images?


I don't have access to oracle cloud ... 

We can support it if: 

1 - you are whiling to make a pull request
2 - you are whiling to maintain that part of the code and be responsible to support every single support request about it.
